Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+## [Unreleased]
```

Comment:
do we need to keep this updated @amshamah419 ?

Reply:
I'm not sure if we are validating these anymore. It may still be required by the content creator step, but I think that issue is fixed. I'll check.